### PR TITLE
Cleanup after truncate packages

### PR DESCRIPTION
**Pull request by lzap**
This pull request includes a cleanup after truncating packages. 

Enables FK with ON DELETE CASCADE.

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
> Enables FK with ON DELETE CASCADE.

I don't see that change :thinking:

File: pkg/models/commits.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ type InstalledPackage struct {
 	Type      string   `json:"type"`
 	Version   string   `json:"version"`
 	Epoch     string   `json:"epoch,omitempty"`
-	Commits   []Commit `gorm:"many2many:commit_installed_packages;save_association:false"`
+	Commits   []Commit `gorm:"constraint:OnDelete:CASCADE;many2many:commit_installed_packages;save_association:false"`
```
Comment (lzap) [inline pkg/models/commits.go:78]: Discussion on order: https://github.com/go-gorm/gorm/issues/4126

**Review by ezr-ondrej — APPROVED** [review]
Thanks :+1:

**Comment by lzap** [comment]
Reminder: after the migration let’s check if the FK does exist.

**Pull request merged**